binaries/daemon/src/spawn.rs

```diff
@@ -302,9 +313,38 @@ pub async fn spawn_node(
         let _ = daemon_tx.send(event).await;
     });
 
+    let node_id = node.id.clone();
     // Log to file stream.
     tokio::spawn(async move {
         while let Some(message) = rx.recv().await {
+            // If log is an output, we're sending the logs to the dataflow
+            if log_output {
+                // Convert logs to DataMessage
+                let array = message.into_arrow();
+
+                let array: ArrayData = array.into();
+                let total_len = required_data_size(&array);
+                let mut sample: AVec<u8, ConstAlign<128>> =
+                    AVec::__from_elem(128, 0, total_len as usize);
+
+                let type_info = copy_array_into_sample(&mut sample, &array);
+
+                let metadata = Metadata::new(uhlc.new_timestamp(), type_info);
+                let output_id = OutputId(node_id.clone(), DataId::from("op/logs".to_string()));
```

Comment:
The `op/logs` output ID only makes sense for runtime nodes with a single operator (see comment above).

Reply:
Agreed!